site/src/pages/TemplatePage/TemplatePage.tsx

```diff
@@ -27,8 +27,9 @@ export const TemplatePage: FC = () => {
       organizationId,
     },
   })
-  const { template, activeTemplateVersion, templateResources } = templateState.context
-  const isLoading = !template || !activeTemplateVersion || !templateResources
+  const { template, activeTemplateVersion, templateResources, templateVersions } =
+    templateState.context
+  const isLoading = !template || !activeTemplateVersion || !templateResources || !templateVersions
```

Comment:
Since the `VersionsTable` has it's own loader that it displays if there are no template versions back yet, can we skip checking it here and not hold the whole page up?

Reply:
Yeah, that is certainly doable.